cmd/manager/main.go

```diff
@@ -40,9 +40,9 @@ var (
 var log = logf.Log.WithName("cmd")
 
 func printVersion() {
-	log.Info("Operator", "version", version.Version)
-	log.Info("Go", "version", runtime.Version(), "os", runtime.GOOS, "arch", runtime.GOARCH)
-	log.Info("Operator-sdk", "version", sdkVersion.Version)
+	log.Info("operator", "version", version.Version)
+	log.Info("go", "version", runtime.Version(), "os", runtime.GOOS, "arch", runtime.GOARCH)
+	log.Info("operator-sdk", "version", sdkVersion.Version)
```

Comment:
This change should happen in the first commit

Reply:
I think this should go in second commit, first commit is for bug fix.